docs/pages/cdp/cdp-Deploying-a-nodejs-application.md

```diff
@@ -53,7 +53,14 @@ CICS TS V5.5 introduced support to run Node.js applications and is required by t
    create : META-INF/cics.xml
    CICS Bundle generated with bundleid "myexpressapp"</pre>
 
-5. Deploy the CICS bundle into CICS.
+5. Confirm that the Node.js SDK is installed on z/OS
```

Comment:
I don't think Node.js is referred to as "Node.js SDK" on other platforms so may seem odd to the reader.
How about "Confirm the Node.js version installed on z/OS is suitable for your application."